frontend/app/components/input/markdown-toolbar.tsx

```diff
@@ -7,11 +7,23 @@ import ItalicIcon from './markdown-toolbar-icons/italic-icon';
 import QuoteIcon from './markdown-toolbar-icons/quote-icon';
 import CodeIcon from './markdown-toolbar-icons/code-icon';
 import LinkIcon from './markdown-toolbar-icons/link-icon';
+import ImageIcon from './markdown-toolbar-icons/image-icon';
 import UnorderedListIcon from './markdown-toolbar-icons/unordered-list-icon';
 import OrderedListIcon from './markdown-toolbar-icons/ordered-list-icon';
 
 interface Props {
   textareaId: string;
+  uploadImages: (files: File[]) => void;
```

Comment:
should be `uploadImages: (files: File[]) => Promise<void>;`

Reply:
I didn't catch, that function returns promise, because this function doesn't have any explicit return and have many return.

Reply:
Yeah, I understand, it just to mark that it's asynchronous, so it won't be confusing that you await it further in "uploadImages" method below.

Reply:
It is two different files, in second case IDE said me, that I called async function. Sad that typescript didn't catch my mistake.